### DISCUSSION TITLE
teams-api

### DISCUSSION THREAD

**Discussion by LaurentJerber** [pycontribs/jira#1210] [Q&A] [answered]
There's another rest api available for jira : teams-api (rest/teams-api/latest/team/?size=100&page=1). I did not see any occurence of it on this github or in the source code.

Is it planned, does it need to be developed ? (I can contribute if needed)

**Comment by studioj** [comment, marked as answer]
no plans until now :-)  maybe add it in the wishlist for https://github.com/pycontribs/jira/discussions/1043
